orchestrate.py

```diff
@@ -69,13 +69,16 @@ def get(self, path=None):
         self.render("loading.html", path=path)
 
 
-class StatsHandler(BaseHandler):
+class APIStatsHandler(BaseHandler):
     def get(self):
         '''Returns some statistics/metadata about the tmpnb server'''
         response = {
                 'available': len(self.pool.available),
                 'capacity': self.pool.capacity,
-                'version': '0.0.1-dev',
+                'version': '0.1.0',
+                'container_image': self.pool.container_config.image,
+                'docker_version': self.spawner.docker_client._docker_client.version(),
+                'docker_containers': self.spawner.docker_client._docker_client.containers()
```

Comment:
Ditto. I got too excited thinking about all the cool things I was gonna do with D3 and this to think about performance.

I'll take it out for now, but I'll give it more thought in a future iteration of the stats page perhaps?